app/controllers/AMIable.scala

```diff
@@ -46,10 +55,7 @@ class AMIable @Inject()(override val amiableConfigProvider: AmiableConfigProvide
         amis <- Attempt.successfulAttempts(amiArns.map(Prism.getAMI))
         amisWithUpgrades = amis.map(Recommendations.amiWithUpgrade(agents.allAmis))
         amisWithInstances = PrismLogic.amiInstances(amisWithUpgrades, instances)
-        amiSSAs = PrismLogic.amiSSAs(amisWithInstances)
-      } yield {
-        Ok(views.html.instanceAMIs(ssa, amisWithUpgrades, PrismLogic.sortSSAAmisByAge(amiSSAs)))
-      }
+      } yield Ok(views.html.instanceAMIs(ssa, amisWithInstances))
```

Comment:
I've spotted that this is done in the template now. I think that if the template only needs them sorted one way, much better to do that here rather than have logic mixed up in the template as well. In an ideal world the template just converts its arguments into HTML.

I realise the template previous had some logic in it (it was a mix of here and there). I'm very sorry about that.